lib/internal/test_runner/runner.js

```diff
@@ -564,6 +565,12 @@ function run(options = kEmptyObject) {
 
   validateString(cwd, 'options.cwd');
 
+  if (existsSync(cwd) === false) {
+    throw new ERR_INVALID_ARG_VALUE('options.cwd', cwd, 'expects an existing directory');
+  } else if (!lstatSync(cwd).isDirectory()) {
+    throw new ERR_INVALID_ARG_VALUE('options.cwd', cwd, 'expects a directory, a file was provided');
+  }
```

Comment:
I would rather not do this. `existsSync()` specifically is a race condition. Just assume that the cwd is a directory and handle any errors. For example, the child process APIs, which also take a `cwd` option don't perform this type of validation.

Reply:
I removed the validation and added a couple of tests to check that `run` handles an incorrect cwd without throwing errors